arangod/Pregel/Conductor/LoadingState.cpp

```diff
@@ -0,0 +1,58 @@
+#include "LoadingState.h"
+
+#include "Pregel/Algorithm.h"
+#include "Pregel/Conductor.h"
+#include "Metrics/Gauge.h"
+#include "Pregel/Conductor/State.h"
+#include "Pregel/PregelFeature.h"
+#include "Pregel/WorkerConductorMessages.h"
+
+using namespace arangodb::pregel::conductor;
+
+#define LOG_PREGEL(logId, level)          \
+  LOG_TOPIC(logId, level, Logger::PREGEL) \
+      << "[job " << conductor._executionNumber << "] "
```

Comment:
Isn't this something that should go into a common header file for Pregel?

Reply:
ah, good point. It can only be used for the conductor states, so I will put it in the conductor namespace in Conductor/State.h

Reply:
maybe it shouldn't be called `LOG_PREGEL` then, but `LOG_PREGEL_CONDUCTOR`?